std_msgs/Header header
geometry_msgs/Pose rel_pose
geometry_msgs/Pose odom_pose
sensor_msgs/PointCloud2 points
geometry_msgs/Pose[] rel_poses